src/helpers/getAllCurrencies.ts

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios'
+
+export const getAllCurrencies = async (setAllCurrencies: any) => {
```

Comment:
getAllCurrencies it's not helper function. This func should be store in service file

Reply:
Okey